### ISSUE TITLE
Visualising 'Requirements' 

### ISSUE THREAD

**Issue by henkvancann** [BlockchainCommons/SmartCustody#9]
https://github.com/BlockchainCommons/SmartCustody/blob/master/Docs/Scenario-Multisig.md#Requirements

Think Amazon? Pictures of the sets you propose.

**Comment by shannona** [comment]
Thanks for the suggestion, Henk.

We've added about 25 illos. Not exactly requirements, but showing how everything is done and what all you produce.

If there's anything else that you think could use an illustration like those we produced, just let us know!

**Comment by imansayadi3** [comment]
🙏🙏Placing bip39 services as a cadre has caused a lot of users who do not have specialized information. For example, (SafePal). They say if you forget the password(extra word or passphrase), we can not help. Therefore, the guarantee and financial security of the user is violated. Bip39 can be placed in the form of additional services and payment for companies and individuals who need it. You sell a house and mention it in the contract if you forget the address and name of the house. The house and the house key have no value. This does not make sense. Endanger.-------- Original message --------From: Shannon Appelcline ***@***.***>Date: Wed, Jun 1, 2022, 04:23To: BlockchainCommons/SmartCustody ***@***.***>Cc: Subscribed ***@***.***>Subject: Re: [BlockchainCommons/SmartCustody] Visualising 'Requirements'  (Issue #9)
Thanks for the suggestion, Henk.
We've added about 25 illos. Not exactly requirements, but showing how everything is done and what all you produce.
If there's anything else that you think could use an illustration like those we produced, just let us know!
